src/opera_rtc_s1_browse/harmony_service.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import tempfile
+from pathlib import Path
+
+import harmony_service_lib
+import pystac
+
+from opera_rtc_s1_browse import create_browse
+
+
+class HarmonyAdapter(harmony_service_lib.BaseHarmonyAdapter):
+
+    def process_item(self, item: pystac.Item, source: harmony_service_lib.message.Source) -> pystac.Item:
+        """
+        Processes a single input item.
+
+        Parameters
+        ----------
+        item : pystac.Item
+            the item that should be processed
+        source : harmony_service_lib.message.Source
+            the input source defining the variables, if any, to subset from the item
+
+        Returns
+        -------
+        pystac.Item
+            a STAC catalog whose metadata and assets describe the service output
+        """
+        self.logger.info(f'Processing item {item.id}')
+
+        with tempfile.TemporaryDirectory() as temp_dir:
+
+            for asset in item.assets.values():
+                if 'data' in (asset.roles or []) and asset.href.endswith('VV.tif'):
```

Comment:
This would be more clear to me:
```suggestion
                if asset.roles is not None and 'data' in asset.roles and asset.href.endswith('VV.tif'):
```